nemo_automodel/components/datasets/llm/hellaswag.py

```diff
@@ -37,13 +46,17 @@ def __init__(self, path_or_dataset, tokenizer, split="train", num_samples_limit=
             split (str, optional): Dataset split to use (e.g., 'train', 'validation'). Defaults to 'train'.
             num_samples_limit (int, optional): Maximum number of samples to load. Defaults to None.
             trust_remote_code (bool, optional): Whether to trust remote code. Defaults to True.
+            shuffle (bool, optional): Whether to shuffle the dataset. Defaults to False.
```

Comment:
do we need shuffling at the dataset level? Distributed sampler should take care of shuffling AFAIK